main.go

```diff
@@ -136,6 +136,12 @@ func main() {
 			EnvVar: "CHECK_INTERVAL,PLUGIN_CHECK_INTERVAL",
 			Value:  time.Duration(1000) * time.Millisecond,
 		},
+		cli.StringSliceFlag{
+			Name:   "create-only",
+			Usage:  "only create resources (do not apply). (optionally as specified as kind/name)",
```

Comment:
Provide a small example here just so it's more clear. I found this a little confusing, I would have expected this to be a bool flag and apply to all resources defined in the files provided to the kd command, rather than the user additionally having to provide the `kind/name` on the end